bookshelf.js

```diff
@@ -0,0 +1,76 @@
+window.bookshelf = (function() {
+	var buttonNode = document.createElement("button");
+	buttonNode.setAttribute("id", 'fetch-books-btn');
+	var textNode = document.createTextNode("Fetch Books");
+	buttonNode.appendChild(textNode);
+	buttonNode.addEventListener("click", fetchBooks);
+	var body = document.getElementsByTagName("body")[0];
+	body.appendChild(buttonNode);
+
+	//function createBtn() {
+		function fetchBooks() {
+			const booksJSON = 'https://raw.githubusercontent.com/codeyourfuture/bookshelf-project/master/books.json'
+			fetch(booksJSON).then(response => response.json()).then(processBooks => {
+				var x = document.createElement("UL");
+				x.setAttribute("id", "myUL");
+				document.body.appendChild(x);
+				processBooks.forEach(function(xz) {
+					var y = document.createElement("LI");
+					y.setAttribute("id", xz.id);
+					var b = document.createElement("button");
+					var bu = document.createTextNode("⬆");
+					b.setAttribute("class", 'moveUp');
+					b.appendChild(bu);
+					 b.addEventListener("click", moveUp);
```

Comment:
you can remove the event listener from the button as you added it to the list which contains the button :)